numpy/lib/format.py

```diff
@@ -158,7 +158,7 @@
 
 import numpy
 import sys
-import io
+from io import BlockingIOError
```

Comment:
Since the code is now python 3 only, you could just replace line 501 to use `io.StringIO` directly. I don't think the current change to this file improves clarity.

Reply:
reverted the change